staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -190,6 +207,21 @@ type Validation struct {
 	Reason *metav1.StatusReason `json:"reason,omitempty" protobuf:"bytes,3,opt,name=reason"`
 }
 
+// AuditAnnotation describes how to produce an audit annotation for an admission request.
+type AuditAnnotation struct {
+	// Key specifies the audit annotation key.
+	// Required.
+	Key string `json:"key" protobuf:"bytes,1,opt,name=key"`
+
+	// ValueExpression represents the expression which is evaluated by CEL to
+	// produce an audit annotation value. The expression must evaluate to either
+	// a string or null value. If the expression evaluates to a string, the
+	// audit annotation is included with the string value. If the expression
+	// evalutes to null the audit annotation will be omitted.
```

Comment:
```suggestion
	// evalutes to null or empty string the audit annotation will be omitted.
```

Reply:
consider what we do if this returns egregiously big values... truncate? skip? document whatever we decide